object-manipulation-1/src/set-value.ts

```diff
@@ -0,0 +1,5 @@
+/* exported setValue */
+function setValue(object: Record<string, any>, key: string, value: any): void {
+  object[key] = value;
+  return undefined;
+}
```

Comment:
When you set the return type to `void` that means that the function is not returning anything, but you have an explicit return for `undefined`. You just don't need the `return` statement for this function.